xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1463,24 +1460,17 @@ namespace VIDEO
       ApplyThumbToFolder(parentDir, art["thumb"]);
   }
 
-  std::string CVideoInfoScanner::GetImage(CFileItem *pItem, bool useLocal, bool bApplyToDir, const std::string &type)
+  std::string CVideoInfoScanner::GetImage(const CScraperUrl::SUrlEntry &image, const std::string& itemPath)
   {
-    std::string thumb;
-    if (useLocal)
-      thumb = CVideoThumbLoader::GetLocalArt(*pItem, type, bApplyToDir);
-
-    if (thumb.empty())
+    std::string thumb = CScraperUrl::GetThumbURL(image);
+    if (!thumb.empty())
     {
-      thumb = CScraperUrl::GetThumbURL(pItem->GetVideoInfoTag()->m_strPictureURL.GetFirstThumb(type));
-      if (!thumb.empty())
+      if (thumb.find("http://") == std::string::npos &&
```

Comment:
please update for the new world of https everywhere. and use StringUtils::StartsWith

Reply:
The find in the line below catches https:// and every other string that has a protocol, this line is unnecessary altogether. I can remove this line and combine the two nested ifs, but I don't see a "Find"-like method in the StringUtils. Is there one somewhere?

Reply:
StringUtils::Contains(..) but really using find for stuff anywhere in string is fine by me. It was the anchoring to front that was most important in my request and startswith conveys that a bit quicker to the reader of code.